modules/images/webp-uploads/load.php

```diff
@@ -227,6 +238,69 @@ function webp_uploads_get_supported_image_mime_transforms() {
 	return (array) apply_filters( 'webp_uploads_supported_image_mime_transforms', $image_mime_transforms );
 }
 
+/**
+ * Creates a resized image with the provided dimensions out of an original attachment, the created image
+ * would be saved in the specified mime and stored in the destination file. If the image can't be saved correctly
+ * a WP_Error would be returned otherwise an array with the file and filesize properties.
+ *
+ * @since n.e.xt
+ * @private
+ * @param int         $attachment_id The ID of the attachment from where this image would be created.
+ * @param array       $size_data An array with the dimensions of the image: height, width and crop.
+ * @param string      $mime The target mime in which the image should be created.
+ * @param string|null $destination_file_name The path where the file would be stored, it should include the extension, when null would use `generate_filename` to create a destination.
```

Comment:
According to WP docs standards:
```suggestion
 * @since n.e.xt
 * @private
 *
 * @param int    $attachment_id         The ID of the attachment from where this image would be created.
 * @param array  $size_data             An array with the dimensions of the image: height, width and crop.
 * @param string $mime                  The target mime in which the image should be created.
 * @param string $destination_file_name The path where the file would be stored, including the extension. If empty, `generate_filename` is used to create the destination file name.
```

The change above also includes my other suggestion to always use a `string` for the last parameter.